tests/acceptance/features/bootstrap/TagsContext.php

```diff
@@ -1463,7 +1468,12 @@ public function sharedByHasNoTags($fileName, $sharingUser) {
 		// The array of tags has a single "empty" item at the start.
 		// If there are no tags, then the array should have just this
 		// one entry.
-		Assert::assertCount(1, $tagList);
+		$numTags = \count($tagList) - 1;
+		Assert::assertCount(
```

Comment:
```suggestion
		Assert::assertEquals(
```
IMO this needs to be `assertEquals` now.
https://drone.owncloud.com/owncloud/core/23422/55/13
```
PHPUnit\Framework\Exception: Argument #2 (No Value) of PHPUnit\Framework\Assert::assertCount() must be a countable or iterable
```